common/sgx/eeid.c

```diff
@@ -495,20 +522,25 @@ oe_result_t verify_eeid(
     uint16_t reported_product_id,
     uint32_t reported_security_version,
     uint64_t reported_attributes,
+    const sgx_attributes_t* reported_sgx_attributes,
+    uint32_t reported_misc_select,
     const uint8_t** base_enclave_hash,
-    const oe_eeid_t* eeid)
+    const oe_eeid_t* eeid,
+    const oe_enclave_size_settings_t* base_image_sizes)
```

Comment:
Does the code extract `base_image_size` from the Attestation Evidence?

Reply:
It is from a customer claim in the Attestation Evidence.

Reply:
That's correct, it's from the base image and communicated via a custom claim.